### PR TITLE
Placed XML in the C# introduction page inside code blocks, and tweaked the comments in the GDScript page

### PR DESCRIPTION
**Pull request by YeldhamDev**

### CODE REVIEW

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 1]
```diff
@@ -151,13 +151,16 @@ as with any project. Many IDEs (such as vs code) can add packages directly. They
...
+
+        <ItemGroup>
+            <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
+        </ItemGroup>
       ...
     </Project>
 
 
-Then using the dotnet command line to restore package, in the project root:
+Then using the dotnet command line to restore package, in the project root::
```
Comment (mhilbrunner) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst]: Double ":"

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 2]
```diff
@@ -151,13 +151,16 @@ as with any project. Many IDEs (such as vs code) can add packages directly. They
...
-        <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
-      </ItemGroup>
+.. code-block:: xml
+   :emphasize-lines: 2
+
+        <ItemGroup>
+            <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
+        </ItemGroup>
       ...
     </Project>
```
Comment (mhilbrunner) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst]: Is this correct? Indentation looks suspect witht hat dangling `</Project>`, but didn't test/build to check

File: getting_started/scripting/c_sharp/c_sharp_basics.rst (hunk 1 above)
Comment (YeldhamDev) [reply getting_started/scripting/c_sharp/c_sharp_basics.rst]: This is to trigger the code block.

File: getting_started/scripting/c_sharp/c_sharp_basics.rst (hunk 2 above)
Comment (YeldhamDev) [reply getting_started/scripting/c_sharp/c_sharp_basics.rst]: This is probably to give an idea to where this part of the code is located.

**Pull request merged**